src/index.html

```diff
@@ -10,8 +11,130 @@
+        </label>
+        <label class="inputBlock">
+          Name:
+          <input type="text" name="name"
+          required  autocomplete="off"
+          >
+        </label>
+        <label class="inputBlock">
+          How old are You?
+          <input type="number" name="age"
+            min="1" max="100"
+            value="12" required
+          >
+        </label>
+        <label class="inputBlock">
+          Full date of birth:
+          <input type="date" name="dateOfbirth"
+            required
+          >
+        </label>
+        <label class="inputBlock">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>Registration</legend>
+        <label class="inputBlock">
+          E-mail:
+          <input type="email" name="email"
+            placeholder="email@example.com" autocomplete="off"
+            required
+          >
+        </label>
+        <label class="inputBlock">
+          Password:
+          <input type="password" name="password"
+            required minlength="8"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <label class="inputBlock">
+          Do you love cats?
+          <label>
+            <input type="radio" name="isLoveCats"
+              value="yes" required
+            >
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="isLoveCats"
+              value="no"  required
+            >
+            No
+          </label>
+        </label>
+        <label class="inputBlock">
+          What is your favorite color?
+          <input type="color" name="favoriteColor"
+            value="#000"
+          >
+        </label>
+        <label class="inputBlock">
+          What time do you go to bed?
+          <input type="time" name="timeToBed"
+            min="16:00" step="1"
+          >
+        </label>
+        <label class="inputBlock">
+          What are your favorite brand of cars?
+          <select name="favoriteCarBrand" size="4"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="inputBlock">
+          How do you rate our work?
+          <input type="range" name="workRate"
+            min="1" max="10"
+            step="1" value="1"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>
+          Additional info:
+        </legend>
+        <label class="inputBlock">
+          Comments:
+          <textarea name="comments" maxlength="2048"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea name="comments" maxlength="2048"
            autocomplete="off"
          ></textarea>
```
To avoid this
![image](https://user-images.githubusercontent.com/97547824/208151553-4f1370e2-8790-4439-b1b4-832068c5c1ef.png)